src/helpers/images.mjs

```diff
@@ -0,0 +1,63 @@
+import { readdir, rename } from "fs/promises";
+import { dirname, join } from "path";
+
+const imageExtensions = /\.(jpg|jpeg)$/i;
```

Comment:
I tested it and it seems to work perfectly!
however, can you change this like to:
const imageExtensions = /\.(jpg|jpeg|png|pdf|gif|heic|webp)$/i;

Reply:
I could do so but if I just changed a .png file for example to a .jpg it should not work?
or should I work on a way to conver any non .jpg image to .jpg, would require using a 3rd party library though.

Reply:
oh i see. maybe we will rely on telling users to use: https://redketchup.io/bulk-image-resizer

okay lets keep it jpg/jpeg

Reply:
Yeah that'd be a better idea, also you can specify in the UI that only jpg images are supported for now for example.
Or make another issue about it later.